docs/usage.md

```diff
@@ -52,12 +50,105 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+   If you have a lot of samples, you may wish to only include one replicate per condition in the samplesheet for assembly, as this may be quicker to run. This command also runs transcriptome quality assessment steps and quantification.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet_assembly.csv \ # samplesheet with only one replicate per sample
+  --outdir <OUTDIR> \
+  -profile docker \
+  --extra_fastp_args='--trim_front1 15 --trim_front2 15' \
+  --remove_ribo_rna \
+  --extra_tr2aacds_args='-MINAA 50' \
+  --busco_lineage= 'arthropoda_odb10' \
+  -resume
+```
+
+4. If you created the transcriptome assembly with only one replicate per condition, you can now run the pipeline with the full samplesheet, path to `--transcript_fasta`, and `--skip_assembly` to quantify the expression of all samples. If you used a samplesheet with all samples in step 3, this step is not needed.
+
+```bash
+nextflow run nf-core/denovotranscript \
+  --input samplesheet.csv \ # full samplesheet
+  --transcript_fasta /path/to/evigene_final_assembly/mrna_fasta_file \
+  --outdir <OUTDIR> \
+  -profile docker \
+  --extra_fastp_args='--trim_front1 15 --trim_front2 15' \
+  --remove_ribo_rna \
+  --skip_assembly \
+  -resume
+```
+
+## Quality Control options
+
+By default, this pipeline checks the quality of the raw reads using FastQC, then performs trimming with fastp. Quality of the trimmed reads is checked again with
+FastQC. You can provide extra arguments to fastp using the `--extra_fastp_args` parameter. You can provide fasta files of adapter sequences to be used for trimming with the `--adapter_fasta` parameter.
+
+Alternatively, you can skip these steps with `--skip_qc` and `--skip_fastp` parameters.
+
+There is also an optional step to remove rRNA reads using SortMeRNA after trimming using the `--remove_ribo_rna` parameter. You can provide a custom ribosomal RNA database with the `--ribo_database_manifest` parameter. By default, the pipeline uses the databases mentioned in `assets/rrna-db-defaults.txt`. You can also save the non-ribo reads with the `--save_non_ribo_reads` parameter. If
+rRNA removal is performed, then FastQC is run again on the non-ribo reads.
+
+## Assembly and redundancy reduction options
+
+The `--assemblers` parameter can be used to specify which assemblers to use. By default, the pipeline uses Trinity and rnaSPAdes (`--assemblers "trinity,rnaspades"`).
+The third assembler option you can add is `"trinity_no_norm"` for running Trinity without normalised reads. Extra parameters can be provided to Trinity using the `extra_trinity_args` parameter.
+
+For rnaSPAdes, the following additional params can be used:
+
+- `--soft_filtered_transcripts` (to include soft filtered transcripts from rnaSPAdes as inputs to Evidential Gene)
+- `--hard_filtered_transcripts` (to include hard filtered transcripts from rnaSPAdes as inputs to Evidential Gene)
+- `--ss` param can be used to set the strand-specific type for rnaSPAdes.
```

Comment:
```suggestion
- `--strand_specific` parameter can be used to set the strand-specific type for rnaSPAdes.
```
or is that parameter name coming from rnaSPAdes?

Reply:
This param is coming from rnaSPAdes, so I will leave it as `ss`